barcode_action/wizard/barcode_action.py

```diff
@@ -0,0 +1,22 @@
+from odoo import fields, models
+
+
+class BarcodeAction(models.TransientModel):
+    _name = 'barcode.action'
+    _inherit = "barcodes.barcode_events_mixin"
+
+    model = fields.Char(
+        required=True, readonly=True
+    )
+    res_id = fields.Integer()
+    method = fields.Char(
+        required=True, readonly=True
+    )
+    state = fields.Selection([
+        ('waiting', 'Waiting'),
+        ('warning', 'Warning')
+    ], default='waiting', readonly=True, )
+    status = fields.Text(
```

Comment:
Fields names are: `state` and `status` ... why not using just one field? :thinking:

Reply:
Well, `status` should be used in order to expand why you have a `state`. At the end, `state` will be used in order to select a color and `status` as the text inside the colored box.